FetchDependency.cmake

```diff
@@ -295,34 +294,24 @@ function(fetch_dependency FD_NAME)
         _fd_run(
           COMMAND "${CMAKE_COMMAND}" -G ${CMAKE_GENERATOR} -S "${SourceDirectory}/${FD_CMAKELIST_SUBDIRECTORY}" -B "${BuildDirectory}" ${ConfigureArguments}
           OUT_COMMAND StepCommand
-          ERROR_VARIABLE ErrorLog
+          ERROR_CONTEXT "Configure failed: "
```

Comment:
The crux of this problem really is that `_fd_run_` needs to be less of a messy hack and actually check the return value of the process run, not just assume failure results in `stderr` being written to. I will address this in a future change though.